README.md

```diff
@@ -28,9 +28,9 @@ This project is still under development. The following features with the check m
 - Rust >= 1.19.0
 - binutils >= 2.22
 - LLVM and Clang >= 3.9 recommended by bindgen
```

Comment:
This is an optional dependency now.

Reply:
maybe add （optional）here

Reply:
Better state in what condition they are required explicitly.